website/project/warmup.rst

```diff
@@ -259,239 +247,264 @@ your code to GitHub.
 You will submit your code through Gradescope. Please note that only one
 team member needs to make the submission (that team member will be given
 the option to specify who else is in their team). When submitting, you
-will be given the option of manually uploading files, or of uploading a
-GitHub repository (we recommend the latter, as this ensures you are
-uploading exactly the files that are in your repository). If you upload
-your repository, make sure you select your
-``2021-design1-team-X`` repository, with “main” as the
+uploading a GitHub repository (we recommend the latter, as this ensures you are
```

Comment:
The sentences in this block seem chopped up (lines 249-252).

Reply:
Good catch, should be fixed now.